ironfish/src/network/peers/connections/webSocketConnection.ts

```diff
@@ -99,8 +99,8 @@ export class WebSocketConnection extends Connection {
         // TODO: any socket that sends invalid messages should probably
         // be punished with some kind of "downgrade" event. This should
         // probably happen at a higher layer of abstraction
-        const message = 'error parsing message'
-        this.logger.warn(`${message} ${event.data.toString()}`)
+        const message = 'Error parsing message'
```

Comment:
Is this an issue on the WebRTCConnection as well?

Reply:
I think we're ok on the WebRTCConnection because the incoming message will be `string | Uint8Array`, but we could make this change there too for consistency?

Reply:
Yep, I think it'd be worth making the same change for consistency at least 👍